src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -161,7 +166,7 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({ disabled, step, setSte
   }, [isMobile, step, setStep])
 
   const onExploreClick = () => {
-    if (isMobile) {
+    if (isMobile && !mobileHelpScreenShown) {
```

Comment:
If user clicks Explore Oasis, goes through the help, clicks Emerald, clicks logo in topbar, clicks Explore Oasis again: then graph stops working

Reply:
Resolved 👍